maintainer/gh_post_status.sh

```diff
@@ -8,4 +8,4 @@ STATUS="$1"
 curl "https://api.github.com/repos/espressomd/espresso/statuses/$GIT_COMMIT?access_token=$GITHUB_TOKEN" \
      -H "Content-Type: application/json" \
      -X POST \
-     -d "{\"state\": \"$STATUS\", \"context\": \"ICP CUDA build\", \"target_url\": \"$URL\"}"
+     -d "{\"state\": \"$STATUS\", \"context\": \"ICP GitLab CI\", \"target_url\": \"$URL\"}"
```

Comment:
Did you also change the status message set by the php script? If not, we should do that before

Reply:
I did not change it yet. Is that /web/kai/status_pending.sh?

We also have bigger fix to fry than fixing typos, yet we still do that. If "ICP CUDA build" continues becoming more general CI and less CUDA, eventually we'll have to change it, so why not do it now.

Reply:
I have now modified the script to automatically pull the context name from this file in the pull request.